packages/webapp/__tests__/SquadFeedPage.tsx

```diff
@@ -275,7 +285,17 @@ describe('squad header bar', () => {
     };
     renderComponent();
     const invite = await screen.findByText('Copy invitation link');
-    invite.click();
+
+    mockGraphQL({
```

Comment:
You should not have to mock this though.
The click should actually execute it no?

Reply:
What do you mean `graphql` mock or `fireEvent`?

Reply:
FireEvent just mimics the graph it's not actually the click event right?
I'm not quite sure this test is viable and it might be a bigger underlying issue rather.

Reply:
I realize me changing something that was not part of the fix might introduce confusion (like click call), usually I try to avoid that. 🙏